tests/integration/test_trampoline.py

```diff
@@ -0,0 +1,90 @@
+import json
```

Comment:
The trampoline is only used for `pixi global`.
Maybe add a folder `tests/integration/global/` and add `test_global` and `test_trampoline` there?

Reply:
Moved it under global module